test/napi/Cargo.toml

```diff
@@ -11,8 +11,9 @@ crate-type = ["cdylib"]
 
 [dependencies]
 once_cell = "1"
+tokio = { version = "1", features = ["rt-multi-thread"] }
 
 [dependencies.neon]
 version = "*"
 path = "../../crates/neon"
-features = ["napi-8"]
+features = ["futures", "napi-experimental"]
```

Comment:
This now means we only test one path through the codebase, when the runtime is implemented differently depending on whether this flag is enabled or disabled. We should probably run our tests both with and without the `"futures"` flag. IMO it's fine if we do this in follow-up work, since our next release will be a beta so the risk is somewhat mitigated.